.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
```

Comment:
I prefer to assign it by project maintainer. At least performance report is not a bug.

Reply:
Okay sure